packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -1325,10 +1328,12 @@ export class IntervalCollection<TInterval extends ISerializableInterval>
 			let deltaProps: PropertySet | undefined;
 			let newInterval: TInterval | undefined;
 			if (props !== undefined) {
-				deltaProps = interval.addProperties(
+				interval.properties ??= new PropertiesManager();
```

Comment:
should be `propertyManager` assigned here, but since both are initialized on creation, better to just remove this line imo

Reply:
there are a couple assignments later in the file as well